js/initcontext.go

```diff
@@ -161,6 +162,11 @@ func (i *InitContext) requireFile(name string) (goja.Value, error) {
 	// First, check if we have a cached program already.
 	pgm, ok := i.programs[fileURL.String()]
 	if !ok || pgm.module == nil {
+		if filepath.IsAbs(name) && runtime.GOOS == "windows" {
+			i.logger.Warnf("you import '%s' with an absolute path, this won't be cross platform and likely will not work if"+
+				" you move the script between machines. Also it might not work on k6 cloud if it doesn't have `file://` in front",
```

Comment:
```suggestion
			i.logger.Warnf("'%s' was imported with an absolute path - this won't be cross-platform and won't work if you move the script"+
				" between machines or run it with `k6 cloud`; if absolute paths are required, import them with the `file://` schema",
```

Reply:
the `file://` will only work for `k6 cloud` or if the do `k6 archive` and then run the archive on another machine. While `if absolute paths are required, import them with the file:// schema` sounds like that will fix everything.

Reply:
Hmm it doesn't to me, and I even change `might not work` from the first sentence to `won't work`. But if you want, change the end to `import them with the file:// schema for slightly better compatibility` :man_shrugging: